pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -149,13 +165,14 @@ def check_metrics(self, logs):
 
         return True
 
-    def on_train_begin(self, logs=None):
+    def on_train_begin(self):
```

Comment:
just curious about the body here, this shall stay even after overwrite...

Reply:
Unfortunately, I don't understand what do you mean :)